xwiki-rendering-api/src/main/java/org/xwiki/rendering/syntax/SyntaxType.java

```diff
@@ -123,6 +125,13 @@
      */
     public static final SyntaxType HTML = register("html", "HTML");
 
+    /**
+     * Browser specific syntaxes.
```

Comment:
Maybe be a bit more specific with something like "Syntaxes that can be rendered diredctly by Browsers."?